build-farm/make-adopt-build-farm.sh

```diff
@@ -208,7 +207,7 @@ if [ "${OPERATING_SYSTEM}" != "aix" ] ; then
     export BUILD_ARGS="${BUILD_ARGS} --create-debug-image"
 fi
 
-echo "$PLATFORM_SCRIPT_DIR/../makejdk-any-platform.sh" --clean-git-repo --jdk-boot-dir "${JDK_BOOT_DIR}" --configure-args "${CONFIGURE_ARGS_FOR_ANY_PLATFORM}" --target-file-name "${FILENAME}" "${TAG_OPTION}" "${OPTIONS}" "${BUILD_ARGS}" "${VARIANT_ARG}" "${JAVA_TO_BUILD}"
+echo "$PLATFORM_SCRIPT_DIR/../makejdk-any-platform.sh --clean-gitrepo --jdk-boot-dir ${JDK_BOOT_DIR} --configure-args ${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --target-file-name ${FILENAME} ${TAG_OPTION} ${OPTIONS} ${BUILD_ARGS} ${VARIANT_ARG} ${JAVA_TO_BUILD}"
```

Comment:
Should that have removed the hyphen from `--clean-gitrepo`? (Or was that an error in my suggestion?)

Reply:
No it shouldn't have. Good catch

Reply:
No it shouldn't have. Good catch